adafruit_emc2101/__init__.py

```diff
@@ -340,12 +376,20 @@ def tach_limit(self):
 
         low = self._tach_limit_lsb
         high = self._tach_limit_msb
-
-        return _FAN_RPM_DIVISOR / ((high << 8) | low)
+        limit = high << 8 | low
+        if limit < 2:
+            print("read tach limit 0")
```

Comment:
Debugging stmt. I'll remove, unless it would be better to throw an exception for this sort of thing. I'm never sure how sparingly to use exceptions, but getting 0 when reading several of the values is definitely abnormal.

Reply:
In general, I would say an exception should be raised when user code should be told something unexpected happen.  I go back to the Zen of Python:

```
Errors should never pass silently.
Unless explicitly silenced.
```

If something has "errored", things depending on it should be made aware upfront.  It's then up to those dependents to determine how to handle the issue, or whether to in fact let them pass.  If this should be an error and the user made aware, it's probably worth stopping execution and raising an error.  If it's just helpful to know, I'd say it's not.  No worries about raising too many though - if it's needed, its needed!